### PR TITLE
Remove deprecated slot syntax

### PR DESCRIPTION
**Pull request by gerhean**
Needed for #1627

```
The current syntax used to provide content to slots has been officially
deprecated, and cannot be used in Vue 3.

Lets use the new slot syntax.
```

Refer [here](https://v2.vuejs.org/v2/guide/components-slots.html?redirect=true#Deprecated-Syntax) for deprecation notice.

Declaring a named slot like in `v-resizer` still uses the same syntax as mentioned [here](https://v2.vuejs.org/v2/guide/components-slots.html?redirect=true#Named-Slots).

### CODE REVIEW

**Review by dcshzj — CHANGES_REQUESTED** [review]
I think `v-resizer.vue` needs to be updated too?

**Comment by gerhean** [comment]
> I think v-resizer.vue needs to be updated too?

Declaring a named slot like in v-resizer still uses the same syntax as mentioned [here](https://v2.vuejs.org/v2/guide/components-slots.html?redirect=true#Named-Slots).

**Review by dcshzj — APPROVED** [review]

File: frontend/src/app.vue [hunk 1]
```diff
@@ -6,65 +6,69 @@
     v-bind:opacity='loadingOverlayOpacity',
     v-bind:is-full-page="true"
   )
-    i.overlay-loading-icon.fa.fa-spinner.fa-spin(slot='default')
-    h3(slot='after') {{ loadingOverlayMessage }}
+    template(slot:default)
+      i.overlay-loading-icon.fa.fa-spinner.fa-spin()
+    template(slot:after)
```
Comment (dcshzj) [inline frontend/src/app.vue:11]: @gerhean Can I double check if `slot` should be changed to `v-slot` here?

Comment (gerhean) [reply frontend/src/app.vue:11]: It's the same, but I suppose  `v-slot` is safer.

**Comment by dcshzj** [comment]
Moving this PR ahead.

**Pull request merged**